voluncheer/templates/volunteer/vol_org_view.html

```diff
@@ -78,5 +88,15 @@ <h2>POSTED OPPORTUNITIES</h2>
         alert('You already registered this event!');
       }
     };
+    // redirect to '/room/<roomInput>/'
+    function toggleChat() {
```

Comment:
why do we need this toggleChat() when we already have toggleChatOrg() and toggleChatVol()?

Reply:
Good point. I'll remove that